CMakeLists.txt

```diff
@@ -196,6 +196,10 @@ if(SFML_OS_WINDOWS)
     if(BUILD_SHARED_LIBS AND SFML_USE_STATIC_STD_LIBS)
         message(FATAL_ERROR "BUILD_SHARED_LIBS and SFML_USE_STATIC_STD_LIBS cannot be used together")
     endif()
+
+    sfml_set_option(SFML_USE_MESA3D FALSE BOOL "TRUE to use the Mesa 3D graphics library for rendering, FALSE to use the system provided library for rendering")
+
+    include(cmake/Mesa3D.cmake)
```

Comment:
Should this `include` be hidden behind an `if(SFML_USE_MESA3D)`?

Reply:
Mesa3D.cmake also includes functionality to remove the associated files if `SFML_USE_MESA3D` evaluates to `FALSE`.

Reply:
What do you mean remove the files? Is it bad if those files are present even if that cache variable is set to `OFF`?

Reply:
You wouldn't be able to use the system OpenGL implementation as long as they are there.